ggml-opencl.cpp

```diff
@@ -175,7 +183,9 @@ void convert_f16(__global half* x, const int ib, const int iqs, float* v0, float
     *v0 = vload_half(0, &x[ib + 0]);
     *v1 = vload_half(0, &x[ib + 1]);
 }
+);
 
+static std::string k_quants_source = MULTILINE_QUOTE(
```

Comment:
maybe out of the scope of this PR, we may want to think to have a separate .cl file to put all OpenCL kernels. Readability is a bit messy now.

Reply:
I agree, but @ggerganov mentioned in the initial OpenCL PR that a single file solution is preferred.